app/services/job_posting_api/parser.rb

```diff
@@ -41,6 +53,18 @@ def parse_job_posting_attributes(job_posting_xml)
 
       private
 
+      def address_attributes(job_posting_xml)
+        address = job_posting_xml.xpath('address')
+        attributes = ADDRESS_FIELDS.transform_values do |xml_key|
+          address.xpath(xml_key).text&.squish.presence
+        end
+
+        attributes[:latitude] &&= attributes[:latitude].to_f
+        attributes[:longitude] &&= attributes[:longitude].to_f
```

Comment:
Since they're optional fields, I have to convert them to floats only if they exist